tests/test_skip.py

```diff
@@ -55,7 +55,7 @@ def test_skip_yes(self):
                    "qubits 7\n"                 \
                    "." + kernel_name + "\n"     \
                    "cnot q[2],q[5]\n"           \
-                   "skip 2"                     \
+                   "skip 2\n"                   \
                    "{ x q[0] | y q[1] }\n"
```

Comment:
Please pull develop into your branch first.  This test case changed in a recent PR. It is strange though why this merge request does not point to that merge but to an older version of the target file.

Reply:
I probably did `git pull` `git merge develop` instead of `git fetch` `git merge origin/develop` so I merged some in-between version I guess. Bad habit.